src/components/CommentBody/web/Blockquote.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
 import { css } from 'glamor'
 
-import colors from '../../../theme/colors'
+import { useColorContext } from '../../Colors/useColorContext'
 import { mUp } from '../../../theme/mediaQueries'
 import { serifRegular14, serifRegular16 } from '../../Typography/styles'
 import { convertStyleToRem } from '../../Typography/utils'
 
 const styles = {
   blockquote: css({
-    backgroundColor: '#f7f7f7',
+    borderLeftWidth: 2,
+    borderLeftStyle: 'solid',
```

Comment:
this defaults to a back border for the not nested case. Should be moved to it's own `blockquoteNested` key